pkg/engine/factories/contextloaderfactory.go

```diff
@@ -85,30 +85,30 @@ func (l *contextLoader) newLoader(
 	if entry.ConfigMap != nil {
 		if l.cmResolver != nil {
 			l := loaders.NewConfigMapLoader(ctx, l.logger, entry, l.cmResolver, jsonContext)
-			return enginecontext.NewDeferredLoader(entry.Name, l)
+			return enginecontext.NewDeferredLoader(entry.Name, l, logr.Logger{})
```

Comment:
There's a logger built in `l`, you should re-use it.

Reply:
To use `l.logger` which is builtin, I would have to rename `l` here: https://github.com/kyverno/kyverno/blob/5577eb87085091aedb30dbce1c768abf2539afb4/pkg/engine/factories/contextloaderfactory.go#L87
Otherwise it creates conflicts here while giving `l.logger` as a parameter: https://github.com/kyverno/kyverno/blob/5577eb87085091aedb30dbce1c768abf2539afb4/pkg/engine/factories/contextloaderfactory.go#L88
Should I rename it to something like `lm` everywhere?